R/foreign-keys.R

```diff
@@ -144,6 +163,8 @@ dm_get_all_fks_impl <- function(dm) {
 #'
 #' @description This function can remove either one reference between two tables, or all references at once, if argument `column = NULL`.
 #' All arguments may be provided quoted or unquoted.
+#' FYI: Mind, that in one of the upcoming updates, compound keys will be supported, and
+#' therefore the syntax of this function will slightly change.
```

Comment:
Same syntax as `dm_add_fk()`, perhaps document on the same page?

Reply:
Ok, and for symmetry reasons I also merged `dm_add_pk()` and `dm_rm_pk()`